projects/1.18/assets/majruszs-progressive-difficulty/majruszs_difficulty/lang/zh_cn.json

```
@@ -73,12 +88,81 @@
   "majruszs_difficulty.undead_army.south": "南方",
   "majruszs_difficulty.subtitle.undead_army.approaching": "亡灵军团：正在袭来",
   "majruszs_difficulty.subtitle.undead_army.wave_started": "新一波敌军已开始进攻",
-  "majruszs_difficulty.effects.haste_tooltip": "破坏方块时给予急迫效果。",
+  "majruszs_difficulty.end_items.bleed_tooltip": "攻击有更大的概率使目标流血",
+  "majruszs_difficulty.end_items.haste_tooltip": "在破坏方块时给予急迫效果",
+  "majruszs_difficulty.end_items.levitation_tooltip": "攻击时有几率对目标施加漂浮效果",
+  "majruszs_difficulty.end_items.till_tooltip": "耕犁 3×3×1 范围的土地",
+  "majruszs_difficulty.sets.end.name": "末地套装",
+  "majruszs_difficulty.sets.end.bonus_2": " [%d/%d] 末地头盔可使所有末影人减少对穿戴者的敌意",
+  "majruszs_difficulty.sets.end.bonus_3": " [%d/%d] 在穿戴者受到攻击时给予穿戴者速度效果",
+  "majruszs_difficulty.sets.end.bonus_4": " [%d/%d] 增加穿戴者2颗心（4点生命值）的最大生命值",
+  "majruszs_difficulty.sets.ocean.name": "海王套装",
+  "majruszs_difficulty.sets.ocean.bonus_2_1": " [%d/%d] 在接触水时增加套装拥有者 20% 的攻击伤害",
+  "majruszs_difficulty.sets.ocean.bonus_2_2": " [%d/%d] 在接触水时增加套装拥有者 30% 的移动速度",
   "majruszs_difficulty.items.undead_helmet": "亡灵军团头盔",
   "majruszs_difficulty.items.undead_chestplate": "亡灵军团胸甲",
   "majruszs_difficulty.items.undead_leggings": "亡灵军团护腿",
   "majruszs_difficulty.items.undead_boots": "亡灵军团靴子",
   "majruszs_difficulty.items.disabled_tooltip": "该物品已被禁用，无法在生存模式中使用！",
+  "majruszs_difficulty.items.advanced_tooltip": "按下Shift键以查看更多细节信息",
+  "majruszs_difficulty.items.set_list_tooltip": "%s (%d/%d)",
+  "majruszs_difficulty.items.set_bonus_tooltip": "增益效果",
   "death.attack.bleeding": "%1$s流血致死",
-  "death.attack.bleeding.player": "%1$s遭受%2$s重创，流血致死"
-}
\ No newline at end of file
+  "death.attack.bleeding.player": "%1$s遭受%2$s重创，流血致死",
+  "death.attack.infested": "%1$s由内及外地被吃掉了",
+  "death.attack.infested.player": "%1$s在与%2$s战斗时被由内及外地吃掉了",
+  "advancements.progressive_difficulty.root.title": "普通模式",
+  "advancements.progressive_difficulty.root.description": "所有事物看上去都十分平静",
+  "advancements.progressive_difficulty.expert_mode.title": "专家模式",
+  "advancements.progressive_difficulty.master_mode.title": "大师模式",
+  "advancements.progressive_difficulty.bleeding.title": "我流血了吗？",
+  "advancements.progressive_difficulty.bleeding.description": "第一次流血",
+  "advancements.progressive_difficulty.bleeding_inflicted.title": "让他们流血",
+  "advancements.progressive_difficulty.bleeding_inflicted.description": "使任意生物流血",
+  "advancements.progressive_difficulty.bleeding_cactus.title": "哎哟！",
+  "advancements.progressive_difficulty.bleeding_cactus.description": "在接触仙人掌后因此流血",
+  "advancements.progressive_difficulty.bleeding_healed.title": "这只是一道划痕！",
+  "advancements.progressive_difficulty.bleeding_healed.description": "使用一片绷带来止血",
+  "advancements.progressive_difficulty.bleeding_healed_other.title": "送给朋友的最好礼物",
+  "advancements.progressive_difficulty.bleeding_healed_other.description": "对其他正在流血的人使用一片金绷带来使他们止血",
```

```suggestion
  "advancements.progressive_difficulty.bleeding_healed_other.description": "对他人使用金绷带",
```
原文：Use a Golden Bandage on someone else